### PR TITLE
chore(vac/sc): update `staking-contract-v1` milestone

### PR DESCRIPTION
**Pull request by 0x-r4bbit**
This aligns the `staking-contract-v1` milestone with the latest quantifiable state, based on the data retrieved from GitHub.

Notice that the status of this milestone has been reset to `0%`. This will correct itself ones current tasks of the milestone are being addressed.

### CODE REVIEW

File: content/vac/sc/g/status/staking-contract-v1.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      'secondaryColor': '#006100', 
+      'tertiaryColor': '#fff' 
+    } 
+  } 
+}%%
+gantt
+  tickInterval 1month
+  dateFormat YYYY-MM-DD 
+  section Status
+    Status Staking Contract: 2023-09-01, 2023-10-31
```
Comment (kaiserd) [inline content/vac/sc/g/status/staking-contract-v1.md]: We should update the end date.
Ideally also add an explanation why it got delayed in the description. (Status has other prios atm)

Comment (0x-r4bbit) [reply content/vac/sc/g/status/staking-contract-v1.md]: Just for me to understand:

```
 section Status
    Status Staking Contract: 2023-09-01, 2023-10-31
```
 
Here we're saying there's a ..... I don't know, "section" called `Status Staking Contract` with a start and end date.
This particular section lives in a `Status` parent section?
Can this be arbitrarily named?
Does this mean we can add multiple items below `section Status` ?

Comment (0x-r4bbit) [reply content/vac/sc/g/status/staking-contract-v1.md]: Also a bit unclear what the new end date is supposed to be. I don't think we actually have a deadline at this point.

Comment (kaiserd) [reply content/vac/sc/g/status/staking-contract-v1.md]: > This particular section lives in a Status parent section? Can this be arbitrarily named?

Yes but we kept this one called Status so far (not to be confused with the org name, it is the status of the milestone.)

The second line is the caption of the milestone.

> Does this mean we can add multiple items below section Status ?

We could. And we could even include several sections see [mermaid doc](https://mermaid.js.org/syntax/gantt.html).
So far, we kept it simple. The Gantt chart just tracks the overall progress of the milestone.

Comment (kaiserd) [reply content/vac/sc/g/status/staking-contract-v1.md]: > Also a bit unclear what the new end date is supposed to be. I don't think we actually have a deadline at this point.

In this case, let comment the gantt out and include it once we know.
Normally, all milestones should have an end date associated, but in cases where projects cannot provide a deadline yet, we may leave it out.

**Pull request merged**